projects/opendr_ws/src/perception/scripts/landmark_based_facial_expression_recognition.py

```diff
@@ -21,6 +21,7 @@
 from vision_msgs.msg import ObjectHypothesis
 from sensor_msgs.msg import Image as ROS_Image
 from opendr_bridge import ROSBridge
+import argparse
```

Comment:
Could you move `argparse` import to near the top where the more generic python imports are?